src/hydro/cache/mysql_cache.py

```diff
@@ -16,9 +18,9 @@ def __init__(self, params=None):
         cache_table = params.get('cache_table', Configurator.MYSQL_CACHE_TABLE)
         cache_db = params.get('cache_db', Configurator.MYSQL_CACHE_DB)
 
-        self.cache = get_cache('django.core.cache.backends.db.DatabaseCache',
-                               **{'LOCATION': cache_table, 'NAME': cache_db})
-        #creating a table if not exist
+        self.cache = DatabaseCache(cache_table, params={'NAME': cache_db})
```

Comment:
Hydro still relies on django's backend classes, though it creates instances of those directly, instead of going thru the wrapper which was removed in 1.9.